assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -333,35 +290,27 @@ const baseSelectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(Object|undefined)} Modules available on the site.
 	 */
-	getModules: createRegistrySelector( ( select ) => ( state ) => {
-		const { modules } = state;
+	getModules( state ) {
+		const { clientDefinitions, serverDefinitions } = state;
 
 		// Return `undefined` if modules haven't been loaded yet.
-		if ( modules === undefined ) {
+		if ( serverDefinitions === undefined ) {
 			return undefined;
 		}
 
-		const registryKey = select( CORE_SITE ).getRegistryKey();
-		if ( registryKey === undefined ) {
-			return undefined;
-		}
-		// Sorting the modules object by order property.
-		const sortedModules = sortByProperty( Object.values( modules ), 'order' );
```

Comment:
I removed the sorting here intentionally as it doesn't make much sense to do when the return value is an object where order shouldn't really be significant. If we want to expose modules in a way where the order is significant, we should add a selector for getting modules as a list where that value would be sorted.

Reply:
@aaemnnosttv This sorting should be maintained in here. Sorting for objects _is_ relevant as we talked about a couple weeks back. Modules should be returned in their correct order by default so that when they're iterated over the order is correct automatically.